### PR TITLE
testcase for resolving an XSD for import located in jar via catalog

### PR DESCRIPTION
**Pull request by tpd-opitz**
This is added as per your suggestion at https://stackoverflow.com/questions/57977802/generate-java-classes-from-xsd-importing-other-xsd-from-jar-provided-via-catalog

The test case fails as expected.  
But unfortunmately I do not get the same error message as with my original setup (which is a little more complex) as described at SO.

Thanks in advance for your effort.

### CODE REVIEW

File: maven-jaxb-pluging-test/importing-xsd/src/main/resources/xsd/importing.xsd [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  targetNamespace="http://my.uri/importingXsd"
+  xmlns:import="http://my.uri/commonDefinitions"
+  xmlns:jxb="http://java.sun.com/xml/ns/jaxb"
+  jxb:version="1.0"
+  elementFormDefault="qualified">
+ <!-- direct relative reference to be resolved - ->
+  <import
+    namespace="http://my.uri/commonDefinitions"
+    schemaLocation="../../../../../common-xsd/src/main/resources/xsd/common.xsd" />
+-->
```
Comment (tpd-opitz) [inline maven-jaxb-pluging-test/importing-xsd/src/main/resources/xsd/importing.xsd:13]: activating the upper `import` and deactivating the lower makes the testcase build successfully.

**Comment by tpd-opitz** [comment]
sometimes reading carefully solves problems...

https://github.com/highsource/maven-jaxb2-plugin/wiki/Using-Catalogs: 

> Due to the [bug](https://java.net/jira/browse/JAXB-942) in the XJC this only works if schemaLocation is not specified. So, unfortunately the following combination does not work at the moment.
> 
> `PUBLIC "http://www.w3.org/1999/xlink" "w3c/1999/xlink.xsd"  
<import namespace="http://www.w3.org/1999/xlink"
  schemaLocation="http://www.w3.org/1999/xlink.xsd" />`

**Comment by highsource** [comment]
I'm sorry this doesn't work. But glad you've figured it out. Thoroughly filing issues often works wonders.

**Comment by tpd-opitz** [comment]
> I'm sorry this doesn't work. But glad you've figured it out. Thoroughly filing issues often works wonders.

There ist nothing to be sorry about, You mentioned the problem in your wiki and I overlooked it. Applying your workaround resolved my problem.

So don't worry, you do a great job! Thanks so much!

**Pull request closed without merging**